features/proxy_config.feature

```diff
@@ -206,3 +206,38 @@ Feature: Proxy configuration
            | release |
            | xenial  |
            | bionic  |
+
+    @series.lts
+    Scenario Outline: Attach command when authenticated proxy is configured
```

Comment:
That's pretty good, but we should also verify if snap and livepatch are answering to this proxy. Furthermore, maybe we could reuse an existing scenario for this test, what do you think ?

Reply:
I'd keep a single scenario for all the authenticated proxy tests for the sake of isolation (then, any authentication issue would break just this one).
I agree that snap and livepatch should be covered, and can be added to this scenario

Reply:
Question:
Is there a reason for the tests in this file targetting the Snap and livepatch proxies are not executed for focal?

Reply:
Yes, livepatch does not support the kvm kernel we run on the lxd instances

Reply:
That's also why we don't have any livepatch test for focal at the moment